Project.toml

```diff
@@ -1,7 +1,7 @@
 name = "HTTP"
 uuid = "cd3eb016-35fb-5094-929b-558a96fad6f3"
 authors = ["Jacob Quinn", "Sam O'Connor", "contributors: https://github.com/JuliaWeb/HTTP.jl/graphs/contributors"]
-version = "0.8.14"
```

Comment:
I'd rather bump this to 0.8.15 since there's nothing breaking here (right?)

Reply:
Ah, yeah I forgot that for 0.x, the minor version is basically seen as major.